README.md

```diff
@@ -101,8 +102,23 @@ The released version of the SDK can be downloaded from https://github.com/Appdyn
     * [C++ SDK and Sample App](https://github.com/Appdynamics/iot-cpp-sdk)
     * [Python Sample Code](https://github.com/Appdynamics/iot-rest-api-sample-apps)
 
+## Miscellaneous
+* Code Coverage 
+    * `jacoco` is integrated in the `build.gradle` file to track code coverage
+    * Generate report
+    ```bash
+    cd <MY_SOURCE_FOLDER>/iot-java-sdk
+    ./gradlew test jacocoTestReport
+    ```
+    * View the report in your favorite browser
+    * For MacOS
```

Comment:
Delete this code snippet. We shouldn't have to provide special instructions for any OS. The same applies to all OS systems: open a file in a browser.